### PR TITLE
It developer post

### PR DESCRIPTION
**Pull request by ErinBecker**
Adds:

* IT developer job posting blog post (dated for publication 21 July)
* IT developer job posting landing page
* IT developer to /jobs/ page

### CODE REVIEW

File: _posts/2022/07/2022-07-21-it-developer.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+time: "09:00:00"
+tags: ["Carpentries Team"]
+---
+
+[The Carpentries is hiring](https://carpentries.org/jobs/)!
+
+The Carpentries is searching for a new full-time Information Technology Developer to join our global, mission-driven organisation. The Information Technology Developer will work with and serve our volunteer community - comprised of 80,000 learners, 3,900 Instructors, 110 Trainers, and 100 member organisations from research institutions around the world. 
+
+Our new Information Technology Developer will join a diverse and engaged team of remote professionals working to spread data-driven research skills around the world. Our team values openness, honesty, fairness, and transparency as the keys to fostering trust. We believe that the individuals who make up our community are the most important part of our organisation and our strongest resource. We strive to make decisions that prioritize people first and empower the growth and success of all who are part of our organisation. We value a growth mindset in all that we do and strive to be always improving and evolving ourselves and our methodologies, being receptive to feedback, responsive, curious, and willing to learn.
```
Comment (tobyhodges) [inline _posts/2022/07/2022-07-21-it-developer.md]:
```suggestion
Our new Information Technology Developer will join a diverse and engaged team of remote professionals working to spread data-driven research skills around the world. Our team values openness, honesty, fairness, and transparency as the keys to fostering trust. We believe that the individuals who make up our community are the most important part of our organisation and our strongest resource. We strive to make decisions that prioritise people first and empower the growth and success of all who are part of our organisation. We value a growth mindset in all that we do and strive to be always improving and evolving ourselves and our methodologies, being receptive to feedback, responsive, curious, and willing to learn.
```

File: pages/it-developer.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+* Working knowledge of R
+* Experience with the administration of self-hosted Shiny applications
+* Experience with the system administration of a cloud infrastructure serving a global audience
+* Experience with UX and front-end libraries (especially Bootstrap)
+* Work authorisation in Europe or South Africa
+
+Our team values openness, honesty, fairness, and transparency as the keys to
+fostering trust. We believe that the individuals who make up our community are
+the most important part of our organisation and our strongest resource. We
+strive to make decisions that prioritize people first and empower the growth and
```
Comment (tobyhodges) [inline pages/it-developer.md]:
```suggestion
strive to make decisions that prioritise people first and empower the growth and
```

File: pages/it-developer.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+teaching practices to teach the core skills for effective computational and data
+work, and we promote reproducible research and open science/open source widely.
+All interactions in The Carpentries community and spaces, both virtual and
+in-person, are governed by our [Code of
+Conduct]({{ site.code_of_conduct_url }}).
+
+
+## Compensation and Benefits
+
+This is a remote position. All qualified applicants are highly encouraged to apply, however, candidates with work authorization in either
```
Comment (tobyhodges) [inline pages/it-developer.md]:
```suggestion
This is a remote position. All qualified applicants are highly encouraged to apply, however, candidates with work authorisation in either 
```

File: pages/it-developer.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+Community Initiatives will make every effort to support the hiring or contracting of international workers, following all local laws related
+to employment and benefits, ensuring a competitive compensation package.
+
+## Equal Employment Opportunity
+
+The Carpentries is dedicated to providing a welcoming and supportive environment
+for all people, regardless of background or identity. We are an equal
+opportunity employer and give consideration for employment to qualified
+applicants without regard to age, race, color, religion, creed, sex, sexual
```
Comment (tobyhodges) [inline pages/it-developer.md]:
```suggestion
applicants without regard to age, race, colour, religion, creed, sex, sexual
```

**Review by tobyhodges — DISMISSED** [review]
small changes to British English spelling - sorry I didn't catch these on my first pass through the posting draft.

**Comment by ErinBecker** [comment]
Thanks for the fixes @tobyhodges! @kmomar  - this is ready to post tomorrow!

**Pull request merged**